src/java.desktop/share/native/libsplashscreen/splashscreen_gif.c

```diff
@@ -279,7 +279,8 @@ SplashDecodeGif(Splash * splash, GifFileType * gif)
                 ImageRect dstRect;
                 rgbquad_t fillColor = 0;                        // 0 is transparent
 
-                if (transparentColor < 0) {
+                if (((colorMap != NULL) && (colorMap->Colors != NULL)) &&
```

Comment:
I can't easily verify that this is an impossible place to be if colorMap == null
so I guess this is OK